mise.toml

```diff
@@ -1,5 +1,5 @@
 [tools]
-node = "20.11.0"
+node = "18.18.0"
```

Comment:
nit: can you make this 18.18.1 so that it aligns with https://github.com/ethereum-optimism/ecosystem/blob/main/.nvmrc?